### PR TITLE
feat(appengine): Update analytics sample to .NET 6

### PR DESCRIPTION
**Pull request by jahjahbin**

### CODE REVIEW

File: appengine/flexible/Analytics/Analytics.Sample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+using System.Net;
+
+public class Program
+{
+    private WebApplication App {get; set;}
```
Comment (amanda-tarafa) [inline appengine/flexible/Analytics/Analytics.Sample/Program.cs:21]: You don't need this, see my suggestion below for best practices. And aslo [Configuration in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0)

File: appengine/flexible/Analytics/Analytics.Sample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        var builder = WebApplication.CreateBuilder(args);
+        App = builder.Build();
+        App.MapGet("/", HandleGetAsync);
+    }
+
+    private async Task HandleGetAsync(HttpContext context)
+    {
+        // [START gae_flex_analytics_track_event]
+        var trackingId = App.Configuration["GA_TRACKING_ID"];
```
Comment (amanda-tarafa) [inline appengine/flexible/Analytics/Analytics.Sample/Program.cs:33]:
```suggestion
    private async Task HandleGetAsync(HttpContext context, IConfiguration configuration)
    {
        // [START gae_flex_analytics_track_event]
        var trackingId = configuration["GA_TRACKING_ID"];

```

**Review by amanda-tarafa — CHANGES_REQUESTED** [review]
One tiny change, otherwise looks good.

We'll also need a reviewer from your side though.

File: appengine/flexible/Analytics/Analytics.Sample/Program.cs (hunk 1 above)
Comment (jahjahbin) [reply appengine/flexible/Analytics/Analytics.Sample/Program.cs:21]: Done, thanks for the suggestion.

**Review by amanda-tarafa — APPROVED** [review]

File: appengine/flexible/Analytics/README.md [hunk 3]
```diff
@@ -3,54 +3,7 @@
 This sample application demonstrates how to track events with Google Analytics
 when running in Google App Engine Flexible Environment.
 
-## Prerequisites
+For detailed steps to deploy this app, check out
+[the Google Cloud
+documentation](https://cloud.google.com/appengine/docs/flexible/integrating-with-analytics?tab=.net).
```
Comment (jinglundong) [inline appengine/flexible/Analytics/README.md:8]: Yup, redirecting to the user guide makes more sense.

**Review by jinglundong — APPROVED** [review]

**Pull request merged**